app/helpers/validate/teachers.spec.js

```diff
@@ -0,0 +1,59 @@
+const assert = require('assert')
+
+const teachers = require('./teachers')
+
+describe('teachers', function () {
+  it('should return empty list when has not errors', function () {
+    const disciplinas = [
+        {
+            teoria: {
+                name: 'Vanessa Kruth',
+                error: null,
+            },
+            pratica: {
+                name: 'Vanessa Kruth',
+                error: null,
+            }
+        },
+        {
+            teoria: {
+                name: 'Isidro',
+                error: null,
+            },
+            pratica: {
+                name: 'Isidro',
+                error: null,
+            }
+        },
+    ];
```

Comment:
Codacy has a fix for the issue: [Extra semicolon.](https://app.codacy.com/gh/ufabc-next/ufabc-next-server/pullRequest?prid=8555383)
```suggestion
    ]
```